packages/widget-chat/src/components/icon/styles.css

```diff
@@ -0,0 +1,6 @@
+.icon {
+  font-family: 'Atlantic Neo Icons';
+  font-size: 18px;
```

Comment:
This size location may need to change if we start using icons in other components. I considered removing it from this PR, but it works for now.